Framework/Geometry/src/Objects/MeshObject.cpp

```diff
@@ -384,10 +381,10 @@ double MeshObject::solidAngle(const Kernel::V3D &observer,
 {
   std::vector<V3D> scaledVertices;
   scaledVertices.reserve(m_vertices.size());
-  for (size_t i = 0; i < m_vertices.size(); ++i) {
-    scaledVertices.push_back(V3D(scaleFactor.X() * m_vertices[i].X(),
-                                 scaleFactor.Y() * m_vertices[i].Y(),
-                                 scaleFactor.Z() * m_vertices[i].Z()));
+  for (auto vertex : m_vertices) {
```

Comment:
I think should be `auto const &vertex` same with others.

Reply:
:+1: I already pushed the change